site/src/docs/components/footer/code.mdx

```diff
@@ -43,7 +43,12 @@ import { Footer, IconFacebook, IconTwitter, IconInstagram, IconYoutube, IconTikt
       <Link label="Contact us" />
       <Link label="Give feedback" />
     </Footer.Utilities>
-    <Footer.Base copyrightHolder="Copyright" copyrightText="All rights reserved" backToTopLabel="Back to top">
+    <Footer.Base
+      copyrightHolder="Copyright"
+      copyrightText="All rights reserved"
+      backToTopLabel="Back to top"
+      logo={<Logo src={logoFi} size="medium" />}
+    >
       <Link label="Link" />
       <Link label="Link" />
       <Link label="Link" />
```

Comment:
Links are not showing, they should be `Footer.Link`. This was my bad from my previous renaming components PR.